base/doc/ltnews41.tex

```diff
@@ -297,134 +327,207 @@ \section{Configurable output routine}
-for package developers to inject special code at defined places that
-can be changed from the outside, for example, to alter the processing.
+\emph{write intro -- say what "tagged PDF" is, please}
+%FMi intro yes, but we don't need to explain tagged PDF ... it is discussed since 2020 in every ltnews
 
-For the tagging support we use a special set of sockets that are
-executed through \cs{UseTaggingSocket} instead of the normal
-\cs{UseSocket} call. This allows us to turn tagging off or on at high
-speed with \cs{SuspendTagging} and \cs{ResumeTagging} without the need
-to individually assign different plugs to the many tagging
-sockets~\cite{41:ltnews39}. This is sometimes necessary, for example,
-when trial typesetting some material several times.
+\subsection{Sockets for tagging support}
+%% Need to explain, at least by a ref, what "tagging" actually is!
 
-In the current release we now offer also dedicated declaration
-commands for the tagging socket (instead of using the underlying
-general socket interface directly), because this better supports the
-special conventions used for tagging sockets. So we now have
-\cs{NewTaggingSocket}, \cs{NewTaggingSocketPlug}, and
+A lot of the tagging support in packages is handled through the socket-and-plug 
+%% hyphenation is used to indicate that this a single concept, 
+%% not two things: "socket" and "plug"
+mechanism that was introduced in \LaTeX{}
+2023-11-01~\cite{41:ltnews38}.
+%
+Sockets offer an easy to use interface
+for package developers to invode variable code at pre-specified places that
+can be changed from the outside by assigning a different plug, for example, to alter the processing.
+
+%FMi: think this is a wrong rewrite 
+%Sockets and plugs offer an interface
+%(that is very easy to use)
+%for package developers to invoke variable code at pre-specified places --- 
+%by simply \enquote{changing the plug}. 
+
+For the tagging support, a specialised set of sockets is available: 
+their plugs are executed
+by using the  \cs{UseTaggingSocket} command, instead of the normal
+\cs{UseSocket} command. This allows tagging to be turned off or on at high
+speed by the commands \cs{SuspendTagging} and \cs{ResumeTagging}, without the need
+to individually reassign plugs to each of the many tagging
+sockets~\cite{41:ltnews39}. This is very useful 
+when there is a need to typeset material several times during trials. 
+
+In the current release we now also offer three dedicated declaration
+commands for these \enquote{tagging sockets}: this works better than 
+directly using the underlying general socket interface. 
+These new commands also better support the
+special conventions used for \enquote{tagging sockets}. They are:  
+\cs{NewTaggingSocket}, \cs{NewTaggingSocketPlug} and
 \cs{AssignTaggingSocketPlug}.
 
 
-\subsection{Promoting PDF 2.0}
+\subsection{Setting the version to PDF 2.0}
+
+%% The logic here is somewhat more complex!
+%% Need to explain why mml gets involved! 
```

Comment:
This is LT News we don't need to explain the tagging project here, only things new in this release